src/scenes/game.js

```diff
@@ -1,17 +1,85 @@
 //Creates phaser class for the scene
 class GameScreen extends Phaser.Scene {
 
-	constructor() {
+	constructor(data) {
 		//the key that references this class, which is used by other functions
 		super('GameScreen')
 	}
 
+	//Basically, this part gets the data from the character select screen
+	init (data)
+	{
+		//This is the sprite object that is passed in
+	    this.choiceLeft = data.playerLeftChoice;
+	    this.choiceRight = data.playerRightChoice;
+	    //This is the array for which frames to start and stop animation. [0] and [1] is for running, [2] and [3] is for attacking.
+	    this.leftFrameArray = data.playerLeftFrames;
+	    this.rightFrameArray = data.playerRightFrames;
+	    //This is the array representing the stats of a player. Strength is [0] and speed is [1]
+		this.leftStrength = data.playerLeftStats[0];
+		this.leftSpeed = data.playerLeftStats[1];
+		this.rightStrength = data.playerRightStats[0];
+		this.rightSpeed= data.playerRightStats[1];
+	}
+	
```

Comment:
this whole section from 10 to 23 is for passing in info from the character select screen to the actual game. I pass in the spritesheet for both characters and their stats.